### ISSUE TITLE
dxf output opens with librecad but not with Autocad

### ISSUE THREAD

**Issue by ebrahimraeyat** [mozman/ezdxf#134]
Hello @mozman . thank you so much for such a quality library. I am developing two softwares that i use your ezdxf library for exporting to dxf. my files open in librecad but did not open with Autocad. I had been produce file in both linux and windows, but none of them open in Autocad. Autocad crach and wants to send error reporting. these are my libraries:

[pyconcrete](https://github.com/ebrahimraeyat/pyconcrete)
[civiltools](https://github.com/ebrahimraeyat/civiltools) 

ezdxf release: (0, 11, 1, 'release')

export dxf file: (I have been change extension of file to .pdf to git allows to upload it.)

[ipe.pdf](https://github.com/mozman/ezdxf/files/4423311/ipe.pdf)

i use this TEMPLATE.dxf file for writing:

[TEMPLATE.dxf](https://github.com/ebrahimraeyat/civilTools/blob/v4/applications/section/TEMPLATE.dxf)

for producing this:

```python
abs_path = 'applications/section'
new_dwg = ezdxf.readfile(abs_path + '/TEMPLATE.dxf')
msp = new_dwg.modelspace()
block = new_dwg.blocks.new(name='ipe14')
block.add_text("ali")
x = [0, 1, 1, 0, 0]
y = [0, 0, 1, 1, 0]
block.add_polyline2d(zip(x, y))
msp.add_blockref('ipe14', (0, 0))
new_dwg.saveas('/home/ebi/alaki/ipe14_1.dxf')
```
with one of the entry text or polyline2d also this happens.
TEMPLATE.dxf itself open with Autocad without error.
thanks.

**Comment by mozman** [comment]
Your are using a DXF R13 file as TEMPLATE.dxf which is upgraded to R2000 by ezdxf, and this process has a bug.

**Comment by mozman** [comment]
Fixed error in bugfix release v0.11.2.

Maybe you can use or test with the latest beta version (actual v0.12b2), I run many tests, but real world usage is sometimes better as seen with this bug.
